style.css

```diff
@@ -2,6 +2,7 @@ html,
 body {
   height: 100%;
   margin: 0;
+  font-family: 'Roboto', sans-serif;
```

Comment:
Can we keep 'Helvetica Neue' as the main? In-fact I already set Roboto as a [fallback](https://github.com/philipp-spiess/this-week-in-react/blob/627157aae02059ae651ce728a60986afc122d600/style.css#L26) so we can just remove this line.

Reply:
Sure, no problem.